hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/HederaWritableStates.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.state;
+
+import com.hedera.node.app.spi.state.WritableStates;
+
+/**
+ * A {@link WritableStates} that can be committed.
+ */
+public interface HederaWritableStates extends WritableStates {
+    boolean isModified();
```

Comment:
You added `isModified` to `WritableQueueStateBase` above, and it already has `commit` - is there a relationship to this interface?

Reply:
I agree, this was all kind of hacky. @tinker-michaelj found a better way to do this that allowed me to unroll this change.